go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containernetworking/plugins
 
-go 1.23
+go 1.22.0
```

Comment:
I agree with you @AkihiroSuda , but how about to change `go 1.22`? Is there any strong reason why we use `1.22.0`?